src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -260,6 +277,48 @@ public function addFluentCommands(Connection $connection, Grammar $grammar)
         }
     }
 
+    /**
+     * Add the alter commands if whenever needed.
+     *
+     * @param  \Illuminate\Database\Connection  $connection
+     * @param  \Illuminate\Database\Schema\Grammars\Grammar  $grammar
+     * @return void
+     */
+    public function addAlterCommands(Connection $connection, Grammar $grammar)
+    {
+        if ($grammar instanceof SQLiteGrammar) {
+            $alterCommands = $grammar->getAlterCommands($connection);
+            $commands = [];
+            $lastCommandWasAlter = false;
+            $hasAlterCommand = false;
+
+            foreach ($this->commands as $command) {
+                if (in_array($command->name, $alterCommands)) {
+                    $lastCommandWasAlter = true;
+                    $hasAlterCommand = true;
+                } else {
```

Comment:
You can merge it into `elseif ($lastCommandWasAlter) {`

Reply:
Thanks!